c2corg_ui/static/js/utils.js

```diff
@@ -196,7 +198,7 @@ app.utils.getTemplate = function(path, $templateCache) {
  */
 app.utils.detectDocumentIdFilter = function(ngeoLocation) {
   // see app.utils.getDoctype() for types definitions
-  var associatedDocTypes = ['w', 'r', 'a'];
+  var associatedDocTypes = ['w', 'r', 'a', 'c'];
```

Comment:
This change is relevant only if we need to pass some article id to an advanced search page (eg. ``waypoints#c=12345``).
I cannot think of a case where we would need this!?